frontend/src/containers/Homepage/Homepage.js

```diff
@@ -40,9 +40,12 @@ const Homepage = ({ location, user, setJwt }) => {
           <div className="row">
             <RecipeSearchList searchTerm={searchTerm} />
           </div>
-          <div className="row">
-            <BottomSignUp />
-          </div>
+          { user ? null
```

Comment:
Can we use `isNil(user)` from [lodash](https://lodash.com/docs/#isNil) here? That'll help clarify the intent and make it less error-prone.